releasenotes/notes/increase-default-max-payload-3147e552384a6fb8.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Increase the Logs Agent default max message payload size from 256 KB to 900 KB.
```

Comment:
Small nit on this: I think `Increase the Logs Agent default max message size from 256 KB to 900 KB.
` would be more accurate. "payload" makes me think about the total HTTP payload size when here we are just changing the default limits for each message.

This could be reflected in the title and description of this PR too.